README.md

```diff
@@ -139,32 +140,11 @@ skipForward()
 skipBackwards()
 ```
 
-### Realtime Audio Manipulation
-
-All audio effects on the player is done through [AVAudioUnit](https://developer.apple.com/documentation/avfoundation/avaudiounit) nodes. These include adding reverb, changing pitch and playback rate, and adding distortion. Full list of effects available [here](https://developer.apple.com/documentation/avfoundation/audio_track_engineering/audio_engine_building_blocks/audio_enhancements).
-
-The effects intended to use are stored in `audioModifiers` as a list of nodes. These nodes are in the order that the engine will attach them to one another.
-
-**Note:** By default `SAPlayer` starts off with one node, an [AVAudioUnitTimePitch](https://developer.apple.com/documentation/avfoundation/avaudiounittimepitch) node, that is set to change the rate of audio without changing the pitch of the audio (intended for changing the rate of spoken word).
-
-#### Important
-All the nodes intended to be used on the playing audio must be finalized before calling `initializeSavedAudio(...)` or `initializeRemoteAudio(...)`. Any changes to list of nodes after initialize is called for a given audio file will not be reflected in playback.
-
-Once all nodes are added to `audioModifiers` and the player has been initialized, any manipulations done with the nodes are performed in realtime. The example app shows manipulating the playback rate in realtime:
-
-```swift
-let speed = rateSlider.value
-if let node = SAPlayer.shared.audioModifiers[0] as? AVAudioUnitTimePitch {
-    node.rate = speed
-    SAPlayer.shared.playbackRateOfAudioChanged(rate: speed)
-}
-```
-
-**Note:** if the rate of the audio is changed, `playbackRateOfAudioChanged` should also be called to update the lockscreen's media player.
+The engine can handle audio manipulations like speed, pitch, effects, etc. To do this, nodes for effects must be finalized before initialize is called. Look at [audio manipulation documentation](#realtime-audio-manipulation) for more information.
 
 ### Lockscreen Media Player
  
-Update and set what displays on the lockscreen's media player when the player is active.
+Update and set what displays on the lockscreen's media player when the player is active. (TODO: Add screenshot)
```

Comment:
I think a single image here will let devs know the capability. Breaks up all the text